linodecli/configuration/__init__.py

```diff
@@ -342,16 +342,19 @@ def configure(
         auth_users = []
 
         if is_full_access:
-            auth_users = [
-                u["username"]
-                for u in _do_get_request(
-                    self.base_url,
-                    "/account/users",
-                    exit_on_error=False,
-                    token=token,
-                )["data"]
-                if "ssh_keys" in u
-            ]
+            users = _do_get_request(
+                self.base_url,
+                "/account/users",
+                token=token,
+                # Allow 401 responses so tokens without
+                # account perms can be configured
+                status_validator=lambda status: status == 401,
+            )
+
+            if "data" in users:
+                auth_users = [
+                    u["username"] for u in users["data"] if "ssh_keys" in u
+                ]
```

Comment:
I think whether the token is with account permission seems has already been accessed here?

https://github.com/linode/linode-cli/blob/4095937df651aa45e7c0e59e99610f29d2342d65/linodecli/configuration/__init__.py#L340